packages/odyssey-storybook/src/contributing/Documentation.stories.mdx

```diff
@@ -0,0 +1,207 @@
+    Reading
+  </dd>
+  <dt>Related Components (optional)</dt>
+  <dd>
+    Points the user toward any related components, possibly alternate solutions.
+  </dd>
+  <dd>
+    We do this in a few places today and it may be beneficial to include in more
+    places.
+  </dd>
+  <dt>Further Reading (optional)</dt>
+  <dd>
+    Again, included in some places, always optional, but could provide
+    additional rationale or education for our users.
+  </dd>
+  <dd>
+    These have been especially helpful when deviating from common dark patterns
+    or legacy solutions.
+  </dd>
+</dl>
+
+## Docs template
+
+```md
+# Landing Gear
+
+This component provides traction and cushion during take-off and landing.
+
+[Figma](http://figma.com/path/to/my-component) - [Source](http://github.com/okta/odyssey/path/to/source)
+
+## Anatomy
+
+**ANATOMY PLACEHOLDER**
+
+## Arguments
+
+<ArgsTable/>
+
+## Behavior (optional)
+
+Landing Gear deploys through doors in the heat shield. This component cannot be retracted once deployed.
+
+## Variants (optional)
+
+Odyssey offers both automatic and manual variants of Landing Gear.
+
+### Manual
+
+Since Landing Gear is deployed through the heat shield, our default variant is Manual control. Unless automated systems have a sufficient success rate, please utilize this variant.
+
+### Automatic
+
+Automatic Landing Gear is deployed once the orbiter has achieved a set proximity to the ground. If your atmospheric density is variable, we recommend using Manual deployment instead.
+
+## States (optional)
+
+### Retracted
+
+In its retracted state, Landing Gear is not visible, being stored behind the heat shield.
+
+### Deployed
+
+In its deployed state, Landing Gear emerges from behind the heat shield and becomes visible. Landings are only possible in this state.
+
+## Usage
+
+Landing Gear should be used when designing any craft with air-to-ground capability.
+
+For most crafts, three sets of Landing Gear are necessary and sufficient. If you are working with particularly large or small crafts, please reach out to the Odyssey team.
+
+### Do
+
+- Utilize Landing Gear when terrestial excursions are rare but expected.
+- Account for both retracted and deployed states in your designs.
+
+### Don't
+
+- Landing Gear should not be included on any craft that will not make terrestial landings.
+- Do not limit your craft to only Deployed or Retracted states. Landing Gear requires both states for full functionality.
+
+## Content Guidelines
+
+When marking left and right Landing Gear controls, treat the nose of your craft as "forward".
+
+Please do not use "port" and "starboard" terminology when referencing controls, as these designations are not useful on some planets.
+
+## Accessibility
+
+Crafts utilizing Landing Gear are required to utilize a nose wheel steering system. Ensure your individual Landing Gear components are associated with the appropriate rudder pedals.
+
+## Responsive Web Design (optional)
```

Comment:
RWD feels like such a dated term. Im wondering if we can do better here (I will also think on it)

Reply:
Same. I did a little hunting around, but RWD seems to still be the standard term.

"Progressive enhancement" might cover a broader range of concerns, but we (Okta) still haven't crossed the RWD hurdle in our products.

LMK if you come up with anything that feels better / more contemporary!